examples/fc_networks.py

```diff
@@ -42,75 +43,72 @@
-print("\nCreated fc-network '%s' successfully.\n  uri = '%s'" % (fc_network['name'], fc_network['uri']))
-
-# Find recently created network by name
-fc_network = oneview_client.fc_networks.get_by('name', 'OneViewSDK Test FC Network')[0]
-print("\nFound fc-network by name: '%s'.\n  uri = '%s'" % (fc_network['name'], fc_network['uri']))
-
-# Update autoLoginRedistribution from recently created network
-fc_network['autoLoginRedistribution'] = False
-fc_network = oneview_client.fc_networks.update(fc_network)
-print("\nUpdated fc-network '%s' successfully.\n  uri = '%s'" % (fc_network['name'], fc_network['uri']))
-print("  with attribute {'autoLoginRedistribution': %s}" % fc_network['autoLoginRedistribution'])
+fc_networks = oneview_client.fc_networks
 
 # Get all, with defaults
 print("\nGet all fc-networks")
-fc_nets = oneview_client.fc_networks.get_all()
+fc_nets = fc_networks.get_all()
 pprint(fc_nets)
 
+# Get the first 10 records
+print("\nGet the first ten fc-networks")
+fc_nets_limited = fc_networks.get_all(0, 10)
+pprint(fc_nets_limited)
+
 # Filter by name
 print("\nGet all fc-networks filtering by name")
-fc_nets_filtered = oneview_client.fc_networks.get_all(filter="\"'name'='OneViewSDK Test FC Network'\"")
+fc_nets_filtered = fc_networks.get_all(filter="\"'name'='Updated FC'\"")
 pprint(fc_nets_filtered)
 
 # Get all sorting by name descending
 print("\nGet all fc-networks sorting by name")
-fc_nets_sorted = oneview_client.fc_networks.get_all(sort='name:descending')
+fc_nets_sorted = fc_networks.get_all(sort='name:descending')
 pprint(fc_nets_sorted)
 
-# Get the first 10 records
-print("\nGet the first ten fc-networks")
-fc_nets_limited = oneview_client.fc_networks.get_all(0, 10)
-pprint(fc_nets_limited)
+# Find recently created network by name
+print("\nGet FC Network by name")
+fc_network = fc_networks.get_by_name(options['name'])
+print(str(fc_network))
+
+if fc_network:
+    print("\nFound fc-network by name: '%s'.\n  uri = '%s'" % (fc_network.data['name'], fc_network.data['uri']))
+else:
+    # Create a FcNetWork with the options provided
+    fc_network = fc_networks.create(data=options)
+    print("\nCreated a fc-network with name: '%s'.\n  uri = '%s'" % (fc_network.data['name'], fc_network.data['uri']))
+
+# Get by uri
+print("\nGet a fc-network by uri")
+fc_nets_by_uri = fc_networks.get_by_uri(fc_network.data['uri'])
+pprint(fc_nets_by_uri.data)
 
-# Get by Id
-if fc_network_id:
+# Update autoLoginRedistribution from recently created network
+data_to_update = {'autoLoginRedistribution': False,
+                  'name': 'Updated FC'}
+fc_network.update(data=data_to_update)
+print("\nUpdated fc-network '%s' successfully.\n  uri = '%s'" % (fc_network.data['name'], fc_network.data['uri']))
+print("  with attribute {'autoLoginRedistribution': %s}" % fc_network.data['autoLoginRedistribution'])
+
+# Adds fcnetwork to scope defined
+print("\nGet scope then add the network to it")
+scope = oneview_client.scopes.get_by_name(scope_name)
```

Comment:
lets update scope resource as well. because it will be used in all other resources

Reply:
No plan to update Scope resource in this version.